### DISCUSSION TITLE
Error using the oh-my-posh command

### DISCUSSION THREAD

**Discussion by patrick-5546** [JanDeDobbeleer/oh-my-posh#1518] [Troubleshoot] [answered]
I get an error when checking the version of Oh My Posh. With Zsh:

``` console
$ oh-my-posh --version
Oh My Posh Error:
 read /home/patrick: is a directory
6.42.3
```

I get this error on version v6.42.1 and later. Error in PowerShell:

``` console
$ oh-my-posh --version
Oh My Posh Error:
 read C:\Users\patrick: The handle is invalid.
6.42.2
```

**Comment by JanDeDobbeleer** [comment]
Goddamnit. I made a mistake.

**Comment by JanDeDobbeleer** [comment, marked as answer]
Is this only when checking the version?

**Reply by patrick-5546** [reply]
No, seems like it occurs every time the oh-my-posh command is used

**Reply by JanDeDobbeleer** [reply]
@patrick-5546 should only happen with an empty config file though. What's your init script like?

**Reply by JanDeDobbeleer** [reply]
@patrick-5546 https://github.com/JanDeDobbeleer/oh-my-posh/pull/1519 (there was an issue, but not on every command unless the --config flag wasn't provided)

**Reply by patrick-5546** [reply]
My config file was based on the atomic theme, but it happened every time I tried the `--version` or `--print-shell` options. Release v6.42.4 fixed all errors I encountered, thanks!
